diff --git a/etc/profile-m-z/zen-browser.profile b/etc/profile-m-z/zen-browser.profile
index fa12d367d8..fa03f4bff3 100644
--- a/etc/profile-m-z/zen-browser.profile
+++ b/etc/profile-m-z/zen-browser.profile
@@ -7,18 +7,18 @@
 include globals.local
 
 # (Ignore entry from disable-common.inc)
-ignore read-only ${HOME}/.zen/profiles.ini
+ignore read-only ${HOME}/.config/zen/profiles.ini
 
 noblacklist ${HOME}/.cache/zen
-noblacklist ${HOME}/.zen
+noblacklist ${HOME}/.config/zen
 
 # uses libgdk-pixbuf and/or glycin - see #6906
 #blacklist /usr/libexec
 
 mkdir ${HOME}/.cache/zen
-mkdir ${HOME}/.zen
+mkdir ${HOME}/.config/zen
 whitelist ${HOME}/.cache/zen
-whitelist ${HOME}/.zen
+whitelist ${HOME}/.config/zen
 
 # Add the following lines to allow access to the .mozilla directory,
 # required by some extensions (like KeePassXC-Browser) to work properly
diff --git a/use-config-folder-zen-browser.patch b/use-config-folder-zen-browser.patch
new file mode 100644
index 0000000000..e69de29bb2
